src/client.toit

```diff
@@ -144,7 +168,11 @@ class Client:
 
   run_:
     while true:
-      packet := transport_.receive --timeout=keep_alive_
+      remaining_keep_alive_us := max
```

Comment:
I think this works okay with negative durations (modulo https://github.com/toitlang/toit/issues/466́). I'd drop the max call.